ui/hooks/useIncrementedGasFees.js

```diff
@@ -56,10 +56,9 @@ function getHighestIncrementedFee(originalFee, currentEstimate) {
  * ).CustomGasSettings} - Gas settings for cancellations/speed ups
  */
 export function useIncrementedGasFees(transactionGroup) {
-  const { primaryTransaction } = transactionGroup;
+  const { primaryTransaction = {} } = transactionGroup;
```

Comment:
It should not be possible for the `primaryTransaction` to be falsy :thinking: We assume it exists all over the place.